<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library Management System</string>
  </property>
  <property name="styleSheet">
   <string notr="true">text-align:center;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="bookList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>331</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QListView" name="bookList_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>331</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Books</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Quicksand Medium</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Members</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>704</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="circulation">
    <property name="title">
     <string>Circulation</string>
    </property>
    <widget class="QMenu" name="menuMember">
     <property name="title">
      <string>Member</string>
     </property>
     <addaction name="actionAdd"/>
     <addaction name="actionDelete"/>
     <addaction name="actionSearch"/>
    </widget>
    <widget class="QMenu" name="menubook">
     <property name="title">
      <string>Book</string>
     </property>
     <addaction name="actionAddBook"/>
     <addaction name="actionEditBook"/>
     <addaction name="actionModifyBook"/>
     <addaction name="actionDeleteBook"/>
     <addaction name="actionIssueBook"/>
     <addaction name="actionReserveBook"/>
     <addaction name="actionSearchBook"/>
    </widget>
    <addaction name="menuMember"/>
    <addaction name="menubook"/>
   </widget>
   <widget class="QMenu" name="menuAquisition">
    <property name="title">
     <string>Aquisition</string>
    </property>
    <widget class="QMenu" name="menuBook_2">
     <property name="title">
      <string>Book</string>
     </property>
     <addaction name="actionDirect_Purchase_2"/>
     <addaction name="actionDonation"/>
     <addaction name="actionGift"/>
    </widget>
    <addaction name="menuBook_2"/>
   </widget>
   <widget class="QMenu" name="menuTechnical">
    <property name="title">
     <string>Technical</string>
    </property>
    <addaction name="actionCatalogue"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <addaction name="actionGeneral"/>
    <addaction name="actionReference"/>
    <addaction name="actionDonated"/>
    <addaction name="actionLibrary_Card"/>
   </widget>
   <addaction name="circulation"/>
   <addaction name="menuAquisition"/>
   <addaction name="menuTechnical"/>
   <addaction name="menuReport"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCatalogue">
   <property name="text">
    <string>Catalogue</string>
   </property>
  </action>
  <action name="actionGeneral">
   <property name="text">
    <string>General</string>
   </property>
  </action>
  <action name="actionReference">
   <property name="text">
    <string>Reference</string>
   </property>
  </action>
  <action name="actionDonated">
   <property name="text">
    <string>Donated</string>
   </property>
  </action>
  <action name="actionLibrary_Card">
   <property name="text">
    <string>Library Card</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionAddBook">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionEditBook">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionModifyBook">
   <property name="text">
    <string>Modify</string>
   </property>
  </action>
  <action name="actionDeleteBook">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionIssueBook">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Issue</string>
   </property>
  </action>
  <action name="actionReserveBook">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reserve</string>
   </property>
  </action>
  <action name="actionSearchBook">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </action>
  <action name="actionDirect_Purchase_2">
   <property name="text">
    <string>Direct Purchase</string>
   </property>
  </action>
  <action name="actionDonation">
   <property name="text">
    <string>Donation</string>
   </property>
  </action>
  <action name="actionGift">
   <property name="text">
    <string>Gift</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
